src/main/resources/createTablescript.sql

```diff
@@ -0,0 +1,14 @@
+	CREATE TABLE users (
```

Comment:
We should not need a script to create a table. Spring can take care of creating tables from entities.